src/concurrency/vector_clock.rs

```diff
@@ -40,7 +40,7 @@ impl From<u32> for VectorIdx {
     }
 }
 
-/// The size of the vector-clock to store inline
+/// The size of the vector clock to store inline
 /// clock vectors larger than this will be stored on the heap
```

Comment:
```suggestion
/// The size of the vector clock to store inline.
/// Clock vectors larger than this will be stored on the heap.
```
I know you didn't write this comment, but if we're touching it anyway we should at least make sure comments that consist of more than one sentence use `.` properly.

Reply:
This change got lost in the rebase.